.circleci/config.yml

```diff
@@ -53,13 +53,30 @@ save-gems-cache: &save-gems-cache
   paths:
     - vendor/bundle
 
-update-fastlane: &update-fastlane
+update-fastlane-ios: &update-fastlane-ios
   name: Update Fastlane
   command: |
     echo "ruby-2.6.4" > ~/.ruby-version
-    bundle install
+    bundle install && bundle update
```

Comment:
@GOVINDDIXIT just a question, bundle update is really needed?
It's added by this PR I can't find the necessity of update all time since we just install and don't use any cache.

Reply:
Sure, will remove it